src/content/_index.en.md

```diff
@@ -18,6 +18,11 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [`subctl`](./operations/deployment/), a friendly deployment tool
 * Support for interconnecting clusters with [overlapping CIDRs](./getting_started/architecture/globalnet/)
 
+{{% notice note %}}
+A few requirements need to be met before you can begin. Check the [Prerequisites](./getting_started/#prerequisites) section for more
```

Comment:
Nir, i think we currently do not have a prerequisite page on the website. Did you miss including the prerequisite as part of this PR? Or are you planning to push a separate PR for that?

Reply:
Yes, we do. Check the link ;)

Reply:
Oh yes, you are right #prerequisites is a link in getting_started. I was thinking its a separate page on its own. Thanks.